release.nix

```diff
@@ -1,7 +1,7 @@
 # 'supportedSystems' restricts the set of systems that we will evaluate for. Useful when you're evaluating
 # on a machine with e.g. no way to build the Darwin IFDs you need!
 { supportedSystems ? [ "x86_64-linux" "x86_64-darwin" ]
-, ifdLevel ? 3
+, ifdLevel ? 2
```

Comment:
I'm not sure to understand the why and how of this change?

Reply:
It's just temporarily on this branch to help hydra evaluate the `bors try` jobs it needs to build.  When we change the way low level derivations in haskell.nix works it can require everything to be rebuilt.  The problem is that hydra wants to instantiate a list of the derivations that need to be built and that in itself requires us to run tools (cabal-install and nix-tools) that will themselves need to be rebuilt.  In theory it would work eventual, but in practice it is a pain because hydra just keeps timing out and we get no visibility on what is going on (because it is rebuilding everything in its `hydra-eval-jobs` stage).  Setting `ifdLevel` to lower values limits excludes the jobs that require these rebuilds (but still builds the tools and various dependencies).

You can see the progress on https://hydra.iohk.io/jobset/Cardano/haskell-nix-bors-trying.

On https://ci.zw3rk.com/project/haskell-nix we have all 4 levels evaluate so if there are issues they sometimes show up there first.

Fortunately most changes do not require messing with `ifdLevel` to get hydra to build them.

Reply:
Thanks for the explanation!